cmd/charm/charmcmd/publish.go

```diff
@@ -35,7 +35,7 @@ The publish command publishes a charm or bundle in the charm store.
 Publishing is the action of assigning one channel to a specific charm
 or bundle revision, so that it can be shared with other users and also
 referenced without specifying the revision. Two channels are supported:
-"stable" and "development", the "stable" channel is used by default.
+"stable" and "development".
```

Comment:
It was correct originally - the stable channel *is* used by default (for publish)